### PR TITLE
Charles marketplace

### PR DESCRIPTION
**Pull request by CharlesYu2000**
stuff

### CODE REVIEW

File: screens/BottomBar.js [hunk 1]
```diff
@@ -60,12 +61,13 @@ class BottomBarBase extends Component{
...
 
 const BottomBar = withFirebase(BottomBarBase);
+const WIDTH_DIVISOR = 13;
 export default BottomBar;
 
 const styles = StyleSheet.create({
     tabs: {
         flexDirection: 'row', 
-        height: theme.sizes.base * 2.5, 
+        height: Dimensions.get('window').width/WIDTH_DIVISOR + 20,
```
Comment (SCAuFish) [inline screens/BottomBar.js]: Nice to avoid magic numbers lol

File: screens/Marketplace.js [hunk 2]
```diff
@@ -93,7 +93,7 @@ class MarketplaceBase extends Component {
   {
       const { type } = this.state; 
       const isActive = type===view.toLowerCase(); 
-      const displayTab = view==='Buying' ? 'Tutor' : 'Student';
+      const displayTab = view!='Buying' ? 'Tutor' : 'Student';
```
Comment (SCAuFish) [inline screens/Marketplace.js]: This is still somehow flipped. Check final comment for more info.

File: screens/Post/NewPost.js [hunk 3]
```diff
@@ -87,7 +87,7 @@ class NewPostBase extends Component{
                     <Block style={styles.inputs}>
                         <Block row space="between" margin={[10, 0]} style={styles.inputRow}>
                             <Block>
-                                <Text style={{ marginBottom: 13, fontWeight: '600', fontSize: 17 }}>What you are looking for?</Text>
+                                <Text style={{ marginBottom: 13, fontWeight: '600', fontSize: 17 }}>I am a</Text>
```
Comment (SCAuFish) [inline screens/Post/NewPost.js]: Nice change here

File: screens/Marketplace.js (hunk 2 above)
Comment (yil578) [reply screens/Marketplace.js]: LOL i think Charles' change is correct cuz "I'm a Tutor", I sell. And as the screenshot shows, if "I'm a Student", I'll see posts that "come from a tutor". @SCAuFish @CharlesYu2000

**Pull request merged**